Mathlib/Topology/UniformSpace/UniformEmbedding.lean

```diff
@@ -14,7 +14,7 @@ Extension of uniform continuous functions.
 -/
 
 
-open Filter Function Set Uniformity Topology
+open Filter Function Set Uniformity UniformSpace Topology
```

Comment:
Would it be easier to not open this NS? Or open it hiding `comap_comap`? If you open it, then you can drop `UniformSpace.` here and there.